e2e/testgames/game/Dockerfile

```
@@ -17,12 +17,36 @@
+COPY assert assert
+
+# Install delve
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
+
+RUN cd e2e/testgames && go mod download
+
+# Set the GOCACHE environment variable to /root/.cache/go-build to speed up build
+ENV GOCACHE=/root/.cache/go-build
+
+# Copy the rest of the source code and build the binary with debugging symbols
+RUN --mount=type=cache,target="/root/.cache/go-build" cd e2e/testgames/game && go build -gcflags="all=-N -l" -v -o /usr/bin/app
```

Hadolint

[warning] Use WORKDIR to switch to a directory

(DL3003)